Yona/Yona/Friends/AddFriendsViewController.swift

```diff
@@ -306,7 +306,7 @@ class AddFriendsViewController: UIViewController, UIScrollViewDelegate, UINaviga
                 let trimmedWhiteSpaceString = number.removeWhitespace()
                 let trimmedString = trimmedWhiteSpaceString.removeBrackets()
                 
-                if trimmedString.validateMobileNumber() == false {
+                if !trimmedString.isValidMobileNumber(){
                     self.displayAlertMessage("", alertDescription:
                         NSLocalizedString("enter-number-validation", comment: ""))
                     
```

Comment:
Do a return here, because otherwise an "add friend" request is done with a wrong mobile number.

Reply:
Updated.